.gdbinit

```diff
@@ -0,0 +1,10 @@
+break Src/main.c:89
```

Comment:
Is it possible to tag a function name or symbol to be the breakpoint? A line number may not be stable.